crates/ironrdp-server/src/capabilities.rs

```diff
@@ -74,6 +74,8 @@ fn virtual_channel_capabilities() -> capability_sets::VirtualChannel {
 
 fn multifragment_update() -> capability_sets::MultifragmentUpdate {
     capability_sets::MultifragmentUpdate {
-        max_request_size: u32::MAX,
+        // FIXME: use an acceptable value for msctc.
+        // What is the actual server max size?
+        max_request_size: 16_777_215,
```

Comment:
cc @awakecoding 

If we can’t figure this out before merging, do you think you can open an issue to track this?